### PR TITLE
Add variance columns to LaTeX diff tables

### LINKED ISSUE #69: Diffs should show change in variation
Sometimes a VM change will result in a benchmark measurement which does not improve the mean steady perf / steady iter, but does improve the overall performance by improving the variation over N pexecs. 

Currently our diffs do not show this. It would be helpful to have another two columns in the table (or similar) to show whether or not each benchmark has improved its variability. First of all, we need a sound statistical way to measure the change in variation. We should not measure an absolute difference in s.d. (or similar), we are interested in relative differences between data sets.

### PR DESCRIPTION
**Pull request by snim2**
Fixes #69

### CODE REVIEW

**Comment by fsfod** [comment]
The new columns generated correctly for me when making a pdf

File: warmup/summary_statistics.py [hunk 1]
```diff
@@ -330,7 +344,9 @@ def convert_to_latex(summary_data, delta, steady_state, diff=None, previous=None
                 time_to_steady = ''
             latex_summary[vm][bmark_name] = {'style': reported_category,
                 'last_cpt': mean_steady_iter, 'last_mean': mean_steady,
-                'time_to_steady_state':time_to_steady}
+                'time_to_steady_state': time_to_steady,
+                'steady_iter_var': steady_iter_var,
+                'steady_time_var': steady_time_var, }
```
Comment (ltratt) [inline warmup/summary_statistics.py]: Stray `, ` at the end?

**Comment by snim2** [comment]
OK to squash?

**Comment by ltratt** [comment]
Yes, please squash.

**Comment by snim2** [comment]
Squashed.

**Pull request merged**